cypress/integration/regression/decision-tree-regression-pt2.spec.ts

```diff
@@ -0,0 +1,255 @@
+    decisionTreesPage.decisionTreeResults.selectReset.select(
+      'Homozygous Recessive/X-linked',
+    )
+    decisionTreesPage.attributesList.replaceButton.click()
+    cy.wait('@decTreeUpload')
+    decisionTreesPage.decisionTreeResults.searchGraphResults
+      .eq(1)
+      .type('Compound_Request')
+    decisionTreesPage.decisionTreeResults.stepCard.countElements(1)
+  })
+
+  it('should create new decision tree | step 27', () => {
+    decisionTreesPage.visit(`/filter?ds=${datasetName}`)
+    decisionTreesPage.decisionTreeResults.addAttribute.click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('aller')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.attributesList.selectAll.contains('Select All').click()
+    cy.intercept('POST', '/app/statunits').as('applyAttributes')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@applyAttributes').its('response.statusCode').should('eq', 200)
+    decisionTreesPage.decisionTreeMenu.createNew.click()
+    decisionTreesPage.decisionTreeMenu.newDecisionTreeNameInput.type(
+      `${decisionTreeName}`,
+    )
+    cy.intercept('POST', '/app/dtree_stat').as('createNewTree')
+    decisionTreesPage.decisionTreeMenu.applyNewTree.click()
+    cy.wait('@createNewTree')
+    decisionTreesPage.decisionTreeResults.anyChangeAlert.checkExists()
+  })
+
+  it('should modify created decision tree | step 28', () => {
+    datasetPage.visit(`/filter?ds=${datasetName}`)
+    cy.intercept('POST', '/app/dtree_set').as('selectList')
+    decisionTreesPage.decisionTreeMenu.selectDecision.first().click()
+    cy.wait('@selectList')
+    cy.intercept('POST', '/app/statunits').as('decTreeUpload')
+    decisionTreesPage.decisionTreeMenu.selectDecision.getFilter(
+      decisionTreeName,
+    )
+    cy.wait('@decTreeUpload')
+    cy.intercept('POST', '/app/dtree_stat').as('stepAfter')
+    decisionTreesPage.decisionTreeResults.optionsMenu.click()
+    decisionTreesPage.decisionTreeResults.addStepAfter.click()
+    cy.wait('@stepAfter')
+    decisionTreesPage.decisionTreeResults.stepCard.countElements(2)
+    decisionTreesPage.decisionTreeResults.addAttribute.eq(1).click()
+    decisionTreesPage.attributesList.searchForAttr.eq(0).type('Min_GQ')
+    decisionTreesPage.decisionTreeResults.graphHeaders.eq(0).click()
+    decisionTreesPage.decisionTreeResults.leftInput.type('10')
+    decisionTreesPage.decisionTreeResults.rightInput.type('100')
+    decisionTreesPage.attributesList.addSelectedAttributes.click()
+    cy.wait('@stepAfter')
+    decisionTreesPage.decisionTreeMenu.decisionActions.click()
+    cy.intercept('POST', '/app/dtree_stat').as('modifyTree')
+    decisionTreesPage.decisionTreeMenu.selectDropdownElem
+      .contains('Modify')
+      .click()
+    decisionTreesPage.decisionTreeMenu.applyNewTree.click()
+    cy.wait('@modifyTree')
+    decisionTreesPage.decisionTreeResults.anyChangeAlert.checkExists()
```

Comment:
Same here we need to check that item is changed actually. I'd check its state before and after the change.

Reply:
hard to check state, added check of the alert